### PR TITLE
Register content

### PR DESCRIPTION
**Pull request by jessebmiller**
Implements register content polywrap methods

### CODE REVIEW

File: polywrap/src/mutation/schema.graphql [hunk 1]
```diff
@@ -10,10 +11,64 @@ type Mutation {
...
+  registerPurchasableNFTContent (
+    connection: Ethereum_Connection
+    licenseManagerAddress: String!
+    nftAddress: String!
+    nftId: BigInt!
+    registrant: String!
+    price: BigInt!
+    sharePercentage: BigInt!
+    data: String
+  ): Ethereum_TxResponse! # ID or maybe the content object?
```
Comment (EzraWeller) [inline polywrap/src/mutation/schema.graphql:23]: We can change the contracts to return this ID from the mint function, right? Maybe we should just do that.

**Review by EzraWeller — APPROVED** [review]
Looks good, few small comments! Fine to merge either way,

File: polywrap/src/mutation/schema.graphql (hunk 1 above)
Comment (jessebmiller) [reply polywrap/src/mutation/schema.graphql:23]: Yeah,  probably should wait until we see it in action a little more I think

File: polywrap/src/mutation/index.ts [hunk 2]
```diff
@@ -29,6 +41,140 @@ export function mintNFT(input: Input_mintNFT): Ethereum_TxResponse {
...
+    ],
+    connection: input.connection,
+    txOverrides: null,
+  })
+
+  return res
+}
+
+
+function handleBase64StringContent(content: string, hash: string | null): ContentInfo {
```
[the remark this answers is not in the document]
Comment (jessebmiller) [reply polywrap/src/mutation/index.ts:67]: That's probably worthwhile

[the remark this answers is not in the document]
Comment (jessebmiller) [reply polywrap/src/mutation/index.ts:67]: Well, I went down that road a little ways and ran into a strange error `[[RuntimeError: table index is out of bounds]]`. This refactor is probably not worth debugging that.

**Pull request merged**